docs/en/integrations/integrations-guidelines.asciidoc

```diff
@@ -0,0 +1,622 @@
+
+[discrete]
+=== Use dashboard-native controls
+
+The **Input controls** visualization type is deprecated in favor of **Controls** embedded into the dashboard itself. The *Controls* dropdown in the Dashboard menu bar should be used. See the {kibana-ref}/add-controls.html[Filter dashboard data with controls] for more information.
+
+[discrete]
+==== Keep Consistent Color
+
+Use color to distinguish categories, represent quantity/density, and highlight data. When using color in this way, be aware that too many colors in a single chart can create noise and hinder quick comprehension.
+
+link:https://elastic.github.io/eui/#/elastic-charts/creating-charts[Elastic UI] can provide guidance for correct color choice.
+Colors provided there for visualization have been tested for accessibility contrast and using them you would be sure to properly serve the biggest audience.
+
+If your dashboard is made to identify specific behaviors it might be interesting to consider a color setting that could help pointing it out. Use a neutral color for generic elements and an accent one for the things that you are looking for.
+
+image::images/colors-in-visualizations.png[Colors in visualizations]
+
+[discrete]
+=== Titles in Visualisations matter
+
+Titles can have a strong visual impact on dashboards, especially when there are a lot of small charts. Two principles can generally be followed:
+
+* Remove unnecessary or repetitive titles when the information is already explained or written within the chart
+* When title is needed make it self explanatory and exhaustive, this way you will be able to remove axis titles and other specifications leaving more space for the charts itself.
+
+image::images/titles-in-visualizations.png[Titles in visualizations]
+
+[[documentation-guidelines]]
+== Documentation guidelines
+
+The goal of each integration's documentation is to:
+
+* Help the reader see the benefits the integration offers and how Elastic can help with their use case.
+Inform the reader of any requirements including system compatibility, supported versions of third-party products, permissions needed, and more.
+* Provide a comprehensive list of collected fields and the data and metric types for each. The reader can reference this information while evaluating the integration, interpreting collected data, or troubleshooting issues.
+* Set the reader up for a successful installation and setup by connecting them with any other resources they'll need.
+* Each integration doc should contain several sections, and you should use consistent headings to make it easier for a single user to evaluate and use multiple integrations.
+
+* <<idg-docs-guidelines-overview>>
+* <<idg-docs-guidelines-datastreams>>
+* <<idg-docs-guidelines-requirements>>
+* <<idg-docs-guidelines-setup>>
+* <<idg-docs-guidelines-troubleshooting>>
+* <<idg-docs-guidelines-reference>>
```

Comment:
This all renders as one ordered list. I think you mean for the second part to break?